src/Files.tsx

```diff
@@ -1,20 +1,33 @@
+import {useGlobalStore} from './hooks/useGlobalStore';
 import * as types from './types';
+import {plural} from './utils';
 
 type FilesProps = {
-  files: types.File[]
+    files: types.FileData[]
 }
 
-export const Files: React.FC<FilesProps> = ({ files }) => {
-  return (
-    <div className="files">
-      <span>+ Files</span>
-      {files.map((file) => (
-        <div id={file.id}>
-        {file.title}
-        <br></br> <br></br>
-        {file.numComments + ' '}
-        Comments
-      </div>))}
-    </div>
-  );
+export const Files: React.FC<FilesProps> = ({files}) => {
+    const globalStore = useGlobalStore();
+
+    return (
+        <div className="files">
+            <span>+ Files</span>
+            {files.map((file) => {
+                const numComments = globalStore.getCommentsForFile(file.id).length;
```

Comment:
The `useGlobalStore` hook helps get arbitrary things from the "already loaded" data. All data for the current project will be loaded at all times, so this data will always be available